mmv1/third_party/terraform/transport/config.go.erb

```diff
@@ -1183,11 +1194,19 @@ func (c *Config) GetCredentials(clientScopes []string, initialCredentialsOnly bo
 
 	log.Printf("[INFO] Authenticating using DefaultClient...")
 	log.Printf("[INFO]   -- Scopes: %s", clientScopes)
+
+	if c.UniverseDomain != "" && c.UniverseDomain != "googleapis.com" {
```

Comment:
Same as above- we should log indicating we've entered this path.

Reply:
Sounds good, will do.